addons/test_mail/tests/test_mail_composer.py

```diff
@@ -1310,6 +1310,39 @@ def test_mail_composer_save_template(self):
         self.assertEqual(template.subject, 'Template Subject')
         self.assertEqual(template.body_html, '<p>Template Body</p>', 'email_template incorrect body_html')
 
+    @users('employee')
+    def test_mail_composer_partner_share_follower(self):
```

Comment:
To define after ``test_mail_composer_recipients``, which holds tests for recipients ^^ maybe  ``test_mail_composer_recipients_status``, something like that ?